cmd/osbuild-composer/main.go

```diff
@@ -38,75 +39,84 @@ func main() {
 		}
 	}
 
-	log.Println("Loaded configuration:")
-	err = DumpConfig(config, log.Writer())
+	logrus.SetOutput(os.Stdout)
+	logLevel, err := logrus.ParseLevel(config.Logging.logLevel)
+
+	if err == nil {
+		logrus.SetLevel(logLevel)
+	} else {
+		log.Println("Failed to set log level", err)
```

Comment:
As this would only happen with an invalid config, this can be logrus.Fatal imo.

Reply:
This would also happen if this field is missing in the config, should we make it a `Warn` ?

Reply:
Well the default is info, and not configuring it should be allowed. So I'd just make it info.